gordo_components/__init__.py

```diff
@@ -51,12 +51,3 @@ def _parse_version(version: str) -> Tuple[int, ...]:
 except Exception:
     warnings.warn(f"Failed to fix absl logging bug {traceback.format_exc()}")
     pass
-
-
-# Set log level, defaulting to INFO
-log_level = os.getenv("LOG_LEVEL", "INFO").upper()
-
-logging.basicConfig(
-    level=getattr(logging, log_level),
-    format="[%(asctime)s] %(levelname)s [%(name)s.%(funcName)s:%(lineno)d] %(message)s",
-)
```

Comment:
Why was this removed?

Reply:
Because, I've failed to move it into a different PR. But the rub is that having it hear isn't good. For one reason if tests with use logging fail, we get an IO error caused by confliction with pytest logging because having it here overrides their logging. Anyhow, I will move this to another PR post haste